### PR TITLE
Bug fix update group roles

### PR DESCRIPTION
**Pull request by JoyalAJohney-KeyValue**

### CODE REVIEW

File: src/authorization/service/group.service.ts [hunk 1]
```diff
@@ -148,23 +149,38 @@ export class GroupService {
     if (!updatedGroup) {
       throw new GroupNotFoundException(id);
     }
+
+    const existingRolesOfGroup = await this.getGroupRoles(id);
```
Comment (steephen6) [inline src/authorization/service/group.service.ts:153]: I can see the same issue exists in the following update calls also. I think we can correct that as well.
updateEntityPermissions
updateGroupPermissions
updateRolePermissions
We are not removing the already present entity permissions, groupPermissions, rolePermissions. Can you please check these also ?

**Review by steephen6 — COMMENTED** [review]
Please check the changes mentioned

File: src/authorization/service/group.service.ts (hunk 1 above)
Comment (steephen6-kv) [inline src/authorization/service/group.service.ts:153]: I can see the same issue exists in the following update calls also. I think we can correct that as well.
updateEntityPermissions
updateGroupPermissions
updateRolePermissions
We are not removing the already present entity permissions, groupPermissions, rolePermissions. Can you please check these also ?

**Review by steephen6-kv — CHANGES_REQUESTED** [review]
Please check the mentioned changes.

File: src/authorization/service/group.service.ts (hunk 1 above)
Comment (JoyalAJohney-KeyValue) [reply src/authorization/service/group.service.ts:153]: Yes, These calls will require the change also. Will update the PR

**Comment by JoyalAJohney-KeyValue** [comment]
PR updated with the fixes for 
**updateEntityPermissions
updateGroupPermissions
updateRolePermissions**
covering their unit tests as well

**Review by steephen6-kv — APPROVED** [review]

**Pull request merged**